test/scripts/jenkins_xpack.sh

```diff
@@ -37,6 +37,11 @@ tar -xzf "$linuxBuild" -C "$installDir" --strip=1
 export TEST_ES_FROM=${TEST_ES_FROM:-source}
 echo " -> Running functional and api tests"
 cd "$XPACK_DIR"
-xvfb-run node scripts/functional_tests --debug --bail --kibana-install-dir "$installDir"
+xvfb-run node scripts/functional_tests --debug --bail --kibana-install-dir "$installDir" --include-tags ciGroup01
 echo ""
+xvfb-run node scripts/functional_tests --debug --bail --kibana-install-dir "$installDir" --include-tags ciGroup02
```

Comment:
I assume this is all here for testing until this and the infra PR's are ready?

Reply:
Yep